tests/e2e/e2e_setup_test.go

```diff
@@ -405,7 +403,10 @@ func (s *IntegrationTestSuite) configureChain(chainId string, validatorConfigs [
 		return
 	}
 
-	s.forkHeight = s.forkHeight - forkHeightPreUpgradeOffset
+	if s.isFork {
```

Comment:
We should add a small comment as to what we're doing here and why.

Reply:
oops, I merged before addressing this. I will address in the next PR. Or @p0mvn if you do an e2e pr before me I can shoot you a comment to add here as well

Reply:
Sounds good. I'll make an issue, whoever opens a PR next can add the comment and close the issue.

Reply:
https://github.com/osmosis-labs/osmosis/issues/1869